examples/api/README.md

```diff
@@ -131,5 +131,5 @@ Tests go into a directory under [test](./test) that matches their location under
 `_test.dart` at the end, like other tests. For instance, a `LayoutBuilder`
 example that resides in [`lib/widgets/layout_builder/layout_builder.0.dart`](
 ./lib/widgets/layout_builder/layout_builder.0.dart) would have its tests in a
-file named [`test/animation/curves/curve2_d.0_test.dart`](
+file named [`test/widgets/layout_builder/layout_builder.0_test.dart`](
```

Comment:
Thanks!  Yes, that's a typo alright.

Also, can you add a paragraph on where the example should be located?
Probably at around line 58 or so:

```
Examples go into a directory under [lib](./lib) that matches the location of the
source file they are linked from, and are named for the symbol they are attached
to, in lower_snake_case, with an index relating to their order within the doc
comment. So, for the `Curve2D` case, since it's in the `animation` package, in a
file called `curves.dart`, and it's the first example, it goes in
`examples/api/lib/animation/curves/curve2_d.0.dart`.
```

Reply:
I'm not sure if where I have the paragraph right now is what you were going for but I just saw that they were discussing the placement of the example file so I added the paragraph to that location. Please let me know what you think.